playkit/src/main/java/com/kaltura/playkit/player/PlayerController.java

```diff
@@ -464,13 +459,17 @@ public void onApplicationResumed() {
 
     @Override
     public void onOrientationChanged() {
+        log.d("onOrientationChanged");
+        if (assertPlayerIstNull("onOrientationChanged()")) {
+            return;
+        }
         player.onOrientationChanged();
     }
 
     @Override
     public void changeTrack(String uniqueId) {
-        if (player == null) {
-            log.w("Attempt to invoke 'changeTrack()' on null instance of the player engine");
+        log.d("changeTrack");
```

Comment:
These should use `log.v` instead of `log.d` so they will only show in verbose mode.